api/test/test_helper/fake_android_plugin.gd

```diff
@@ -24,11 +24,25 @@
 class_name FakeAndroidPlugin
 extends Object
 
+signal removePackSuccess
+signal removePackError
+signal showCellularDataConfirmationSuccess
+signal showCellularDataConfirmationError
+
 var _asset_location_store : Dictionary
 var _asset_pack_location_store : Dictionary
 
 var _asset_pack_states_store : Dictionary
 
+var _show_confirmation_thread : Thread
```

Comment:
This class is starting to have a lot of groups of fields/methods with different prefixes, and I imagine there will be more as we add other methods. Is there a way to break those elements up into their own classes and have fake_android_plugin reference them?

For example these fields/methods could be broken out into another class `fake_cellular_confirmation_handler`:
````
var _show_confirmation_thread : Thread
var _show_confirmation_success : bool
var _show_confirmation_result : int
var _show_confirmation_error : Dictionary
func set_show_confirmation_response()
````

Reply:
Feel free to pick a different name than `fake_cellular_confirmation_handler `.

Reply:
btw this refactoring can come in a later PR.

Reply:
Yeah I think refactoring these variables into their own classes is definitely a better design decision. I added a TODO comment for this suggestion and will work on this in next PR.